homeassistant/components/shelly/utils.py

```diff
@@ -664,10 +647,7 @@ def async_remove_shelly_rpc_entities(
 
 def get_virtual_component_ids(config: dict[str, Any], platform: str) -> list[str]:
     """Return a list of virtual component IDs for a platform."""
-    component = VIRTUAL_COMPONENTS_MAP.get(platform)
-
-    if not component:
-        return []
```

Comment:
We don't need to use `get` here since we use this only in places that has mapping